docs/tutorial/object_decomp_example.md

```diff
@@ -0,0 +1,254 @@
+
+![Showing the head limb's display list in Z64Utils](images/z64utils_dns_limb_dlist.png)
+
+Another way is to simply check `object_dns.c`. Each limb lists its own display list like this:
+```c
+StandardLimb gKingsChamberDekuGuardHeadLimb = { 
+    { 0, 1300, 0 }, KINGS_CHAMBER_DEKU_GUARD_STALK_LIMB - 1, KINGS_CHAMBER_DEKU_GUARD_LEFT_FOOT_LIMB - 1,
+    object_dns_DL_0018B8
+};
+```
+
+Either way you go about it, you should be able to name all the limb display lists like so:
+```xml
+<DList Name="gKingsChamberDekuGuardRightFootDL" Offset="0x1640" />
+<DList Name="gKingsChamberDekuGuardLeftFootDL" Offset="0x16F0" />
+<DList Name="gKingsChamberDekuGuardSnoutDL" Offset="0x17A0" />
+<DList Name="gKingsChamberDekuGuardHeadDL" Offset="0x18B8" />
+<DList Name="gKingsChamberDekuGuardStalkDL" Offset="0x19B8" />
+<DList Name="gKingsChamberDekuGuardEyesDL" Offset="0x1A50" />
+<DList Name="gKingsChamberDekuGuardTorsoDL" Offset="0x1AD8" />
+<DList Name="gKingsChamberDekuGuardLeftMustacheDL" Offset="0x1B90" />
+<DList Name="gKingsChamberDekuGuardRightMustacheDL" Offset="0x1C20" />
+<DList Name="gKingsChamberDekuGuardLeftLeafDL" Offset="0x1CB0" />
+<DList Name="gKingsChamberDekuGuardCenterLeafDL" Offset="0x1D40" />
+<DList Name="gKingsChamberDekuGuardRightLeafDL" Offset="0x1DD8" />
+```
+
+Run `./extract_assets.py -s objects/object_dns` once again, since it will help in the next step to have all of our display lists named.
+
+### Step #6: Naming remaining textures
+
+With every display list named, it's now a lot easier to name the remaining textures. In the `assets/objects/object_dns/` folder, you can see all the textures in the object as various PNG files. For some of the textures, just looking at them will give you a good idea as to what they should be named. For other textures, it may help to see how the texture is used in the object's display lists. Let's take a look at `object_dns_Tex_002868`, which is only used in one display list:
+```c
+Gfx gKingsChamberDekuGuardSnoutDL[] = {
+    [...]
+    gsDPLoadTextureBlock(object_dns_Tex_002868, G_IM_FMT_RGBA, G_IM_SIZ_16b, 8, 8, 0, G_TX_MIRROR | G_TX_CLAMP,
+                         G_TX_MIRROR | G_TX_CLAMP, 3, 3, G_TX_NOLOD, G_TX_NOLOD),
+    [...]
+};
+```
+
+Since `tex_002868.rgba16.png` looks like the inside of the Deku Guard's mouth, and since it's *only* used in the snout display list, you can probably name this something like `gKingsChamberDekuGuardMouthTex`. But rather than guessing, we can confirm it by editing the texture ourselves and then viewing it in-game. Let's change `tex_002868.rgba16.png` to this:
+
+![Our custom mouth texture](images/dns_custom_texture.png)
+
```

Comment:
I think it would be beneficial show/explain exactly how you changed the texture. In particular the first bullet point of the investigative modding seems unclear to actually how to do it.

Reply:
I'm probably the worst person to ask about this sort of thing because I use Paint like an idiot, lol. But I took a shot at explaining it in both the investigative modding section and in here.